### ISSUE TITLE
Some error happened, after upgrade 

### ISSUE THREAD

**Issue by juaresz** [canton7/SyncTrayzor#202]
```
Version: 1.1.1.231; Variant: Installed; Arch: X86
Path: C:\Program Files\SyncTrayzor\SyncTrayzor.exe
System.OperationCanceledException: A operação foi cancelada.
   em System.Threading.CancellationToken.ThrowOperationCanceledException()
   em SyncTrayzor.Services.Conflicts.ConflictFileManager.FindConflictsImpl(String basePath, SlimObservable`1 subject, CancellationToken cancellationToken) na C:\projects\synctrayzor\src\SyncTrayzor\Services\Conflicts\ConflictFileManager.cs:linha 128
   em SyncTrayzor.Services.Conflicts.ConflictFileManager.<>c__DisplayClass8_0.<FindConflicts>b__0() na C:\projects\synctrayzor\src\SyncTrayzor\Services\Conflicts\ConflictFileManager.cs:linha 110
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em SyncTrayzor.Utils.SlimObservableExtensions.<SubscribeAsync>d__1`1.MoveNext() na C:\projects\synctrayzor\src\SyncTrayzor\Utils\SlimObservable.cs:linha 139
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em SyncTrayzor.Services.Conflicts.ConflictFileWatcher.<ScanFoldersAsync>d__31.MoveNext() na C:\projects\synctrayzor\src\SyncTrayzor\Services\Conflicts\ConflictFileWatcher.cs:linha 204
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em SyncTrayzor.Services.Conflicts.ConflictFileWatcher.<Reset>d__26.MoveNext() na C:\projects\synctrayzor\src\SyncTrayzor\Services\Conflicts\ConflictFileWatcher.cs:linha 97
--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---
   em System.Runtime.CompilerServices.AsyncMethodBuilderCore.<ThrowAsync>b__4(Object state)
   em System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   em MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
```

[SyncTrayzor.zip](https://github.com/canton7/SyncTrayzor/files/88735/SyncTrayzor.zip)

**Comment by canton7** [comment]
Dup of #200. I'll push out 1.1.2 shortly, hopefully today.

**Comment by canton7** [comment]
Fixed in v1.1.2.
